rascal-vscode-extension/package.json

```diff
@@ -47,8 +47,22 @@
       {
         "command": "rascalmpl.importModule",
         "title": "Start Rascal Terminal and Import this module"
+      },
+      {
+        "command": "rascalmpl.startDebuggerForRepl",
+        "title": "Start Rascal debugger for REPL",
+        "icon": "$(debug)"
       }
     ],
+    "menus": {
+      "view/item/context": [
+        {
+          "command": "rascalmpl.startDebuggerForRepl",
```

Comment:
similar here, if we move the command declaration to an annotation on the list item node, we can get away with the when class I think?